modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -124,6 +128,12 @@ const DEFAULT_PROPS: Props = {
   maxRequests: 64,
 
   maximumMemoryUsage: 32,
+  // Maximum number limit of tiles selected for show
```

Comment:
Use `/** */` then vscode can show such comments in the tooltips.

Reply:
Done